docs/apm-package/input-apm.asciidoc

```diff
@@ -0,0 +1,67 @@
+:input-type: apm
+
+[[input-apm]]
+== APM input settings
+
+experimental::[]
+
+IMPORTANT:
```

Comment:
Is it the following paragraph that's important? It's not obvious to me looking at the rendered page whether that was intended or if there's missing text, comparing to other pages.

Reply:
Ah, good catch! I had an admonition paragraph here but removed the text in https://github.com/elastic/apm-server/pull/4928/commits/3e0377bc587076019490f306be1d858d78514533 because there were three at the top of the page: experimental, tip, and important. It was just a little too busy. Looks like I forgot to remove the admonition tag.